apps/hs/qua-server/src/Foundation.hs

```diff
@@ -482,3 +487,13 @@ lastScenarioProblemId = getVal <$> rawSql query []
     query = unlines
           ["SELECT max(id) FROM scenario_problem;"
           ]
+
+
+maybeEnroll :: Email -> Maybe Text -> Handler ()
+maybeEnroll email msId =
+    case msId of
+        Nothing -> pure ()
+        Just i -> do
+             $(logDebug) $ "Enrolling new user with email " <> email <> " in scenario problem " <> i
+             runDB $ pure () -- TODO what here?
```

Comment:
This is where we hook into the enrollment, but I'm not sure what exactly to do. I couldn't figure it out from the DB models.